environments/dev/main.jsonnet

```diff
@@ -87,6 +87,17 @@ local obs = (import '../base/observatorium.jsonnet') + {
   },
 };
 
+local tls_secret = (import '../../components/observatorium-api-tls-secret.libsonnet') + {
```

Comment:
Rather than being a separate file that need to be imported, the TLS secret should be automatically appended to the observatorium manifests when you add the withTLS mixin to the API. There should be no need to explicitly add another set of manifests or input another libsonnet file

Reply:
done